packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js

```diff
@@ -6822,6 +6822,48 @@ QUnit.module('performance', () => {
         assert.ok($.isFunction(filter), 'filter is function');
     });
 
+    QUnit.test('tag labels should be correctly displayed with valueExpr as function and hideSelectedItems enabled (T1234032)', function(assert) {
+        const dataSource = [
+            { id: 1, scheme: 'schema 1', name: 'item1' },
+            { id: 2, scheme: 'schema 2', name: 'item2' },
+            { id: 3, scheme: 'schema 3', name: 'item3' },
+            { id: 4, scheme: 'schema 4', name: 'item4' },
```

Comment:
[minor] i believe 3 items is enought, lest not create redundant data for tests

Reply:
[875d1f5](https://github.com/DevExpress/DevExtreme/pull/27521/commits/875d1f5dad71b09a9bfe80286d68cbc533e78732)